inlang/source-code/detect-formatting/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "@inlang/detect-formatting",
```

Comment:
@NilsJacobsen can we name it @inlang/detect-json-formatting for more explicitness also when people search for the package?

Reply:
Sure, you are right that feels more self-explaining.